xbmc/video/VideoInfoScanner.cpp

```diff
@@ -500,11 +500,18 @@ namespace VIDEO
 
         auto eventLog = CServiceBroker::GetEventLog();
         if (eventLog)
-          eventLog->Add(EventPtr(new CMediaLibraryEvent(
-              mediaType, pItem->GetPath(), 24145,
-              StringUtils::Format(g_localizeStrings.Get(24147), mediaType,
-                                  URIUtils::GetFileName(pItem->GetPath())),
-              pItem->GetArt("thumb"), CURL::GetRedacted(pItem->GetPath()), EventLevel::Warning)));
+          if(info2->Content() == CONTENT_TVSHOWS)
+            eventLog->Add(EventPtr(new CMediaLibraryEvent(
+            mediaType, pItem->GetPath(), 24145,
+            StringUtils::Format(g_localizeStrings.Get(24147), mediaType,
+            CURL::GetRedacted(pItem->GetPath())),
+            EventLevel::Warning)));
+          else if (info2->Content() == CONTENT_MOVIES || info2->Content() == CONTENT_MUSICVIDEOS)
```

Comment:
I saw @Montellese mentioned the duplication a lot of the eventlog call. Could something like the following be an option

```
diff --git a/xbmc/video/VideoInfoScanner.cpp b/xbmc/video/VideoInfoScanner.cpp
index 6b7ab281e5..8ac9c5cc6d 100644
--- a/xbmc/video/VideoInfoScanner.cpp
+++ b/xbmc/video/VideoInfoScanner.cpp
@@ -500,11 +500,21 @@ namespace VIDEO
 
         auto eventLog = CServiceBroker::GetEventLog();
         if (eventLog)
+        {
+          std::string itemlogpath;
+          if(info2->Content() == CONTENT_TVSHOWS)
+          {
+            itemlogpath = CURL::GetRedacted(pItem->GetPath());
+          }
+          else
+          {
+            itemlogpath = URIUtils::GetFileName(pItem->GetPath());
+          }
           eventLog->Add(EventPtr(new CMediaLibraryEvent(
               mediaType, pItem->GetPath(), 24145,
-              StringUtils::Format(g_localizeStrings.Get(24147), mediaType,
-                                  URIUtils::GetFileName(pItem->GetPath())),
-              pItem->GetArt("thumb"), CURL::GetRedacted(pItem->GetPath()), EventLevel::Warning)));
+              StringUtils::Format(g_localizeStrings.Get(24147), mediaType, itemlogpath),
+              EventLevel::Warning)));
+        }
       }
 
       pURL = NULL;
```

It does the single call, but does your check of CONTENT_TVSHOWS and uses the filepath as intended. But you dont have to duplicate the rest of the static info in the eventLog->Add() call. The else also continues with the existing catch all in case there is some other CONTENT_* type that doesnt fall into those 3 i mentioned above. That may not even be a concern, but better safe i guess.